adr/0014-home-assistant-supervised.md

```diff
@@ -0,0 +1,78 @@
+# 0012. Installation method: Home Assistant Container
+
+Date: 2020-06-03
+
+## Status
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+This installation method provides the full Home Assistant experience on a regular operating system. This means, all components from the Home Assistant method are used, except for the Home Assistant Operating System. We don’t support using the host system for other applications than the Supervisor. You need to understand that the Supervisor is not just an application, it is a full appliance.
+
+By not using the Home Assistant Operating System, the user is responsible for making sure that all required components are installed and maintained. Required components and their versions will change over time. We are not responsible for issues that come up with your host or wrong configuration.
+
+This method is considered advanced and should only be used if one is an expert in managing a Linux operating system, Docker and networking.
+
+### Supported Operating System, System dependencies and versions
+
+Docker CE (Community Edition) is the only supported containerization method for Home Assistant Supervised. We only support FSH 3.0 on the host file system.
+
+- Docker CE >= 19.03
+- Systemd >= 239
+- NetworkManager >= 1.18.0
+- Avahi >= 0.7
+- AppArmor == 2.13.x
+- Debian Linux Debian 10 aka Buster
+
+Only the above-listed version of Debian Linux is supported for running this installation method. When a new major version of Debian is released, the previous major version is dropped, with a deprecation time of 4 months. An exception to this rule occurs if the new version does not solve the requirements of the Supervisor.
+
+### Additional supported conditions
+
+This installation method can easily be broken if one manages the operating system incorrectly. Therefore, the following additional conditions apply:
+
+- The operating system is dedicated to running Home Assistant Supervised.
+- All system dependencies are installed according to the manual.
+- No additional software, outside of the Home Assistant ecosystem, is installed.
```

Comment:
This one will need some extra explaining. It will be along the lines: 

The reason for this requirement is that the Supervisor turns your system into an appliance: a system with the specific task of running Home Assistant. This is not an "app" that you install. Your system becomes Home Assistant Supervised.

This means that the supervisor will expect to be able to clean and correct the system to ensure smooth operations. Examples:

 - If a config can't be parsed, we should be able to reset it to defaults.
 - cleanup tasks will remove all Docker containers that are not running